feat: L1 P2 19-1

JesusHuayhua opened the pull request.

JesusHuayhua committed:
feat: L1 P2 19-1

ManuelLoaizaV commented on laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp:
```diff
@@ -0,0 +1,195 @@
+    std::cout << "Ingresar el tamaño de la cuadricula: \n";
+    int t, cant;
+    std::cin >> t;
+    char **tablero, **word, res[3];
```
`word` is an array of strings, so its name should be `words`.

ManuelLoaizaV replied:
Can `res` be more descriptive? I think is a result, but not so sure what it contains.

ManuelLoaizaV commented on laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp:
```diff
@@ -0,0 +1,195 @@
+    for (int i = 0; i < t; i++) {
+        word[i] = new char[t + 1];
+    }
+    //Lectura de las palabras
```
Try to be more consistent. In the previous comment you have used spacing between the slashes and the text.
```suggestion
    // Lectura de las palabras
```

ManuelLoaizaV commented on laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp:
```diff
@@ -0,0 +1,195 @@
+    PopulateGameBoard(t, tablero);
+    ShowGameBoard(t, tablero);
+    std::cout << "Ingrese la cantidad de palabras a buscar\n";
+    std::cin >> cant;
```
`cnt` or `cantidad` might be better but `cant` is in an existential crisis.

ManuelLoaizaV commented on laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp:
```diff
@@ -0,0 +1,195 @@
+        std::cin >> *(word + i);
+    }
+
+    //Busqueda de las palabras
```
The same in here.

ManuelLoaizaV reviewed (changes requested):
Thanks, @JesusHuayhua.

JesusHuayhua committed:
Update L1/P2/19-1

Change name of variables and return in the search functions right and down the word

JesusHuayhua commented:
Hi @ManuelLoaizaVasquez , I change name of variables, and return in the search functions right and down
